### PR TITLE
Arguments

### PR DESCRIPTION
**Pull request by emmanuelle**
This PR should only be merged and deployed when 4.2.2 is out, including https://github.com/plotly/plotly.py/pull/1768, but we can start discussing it here.

Doc upgrade checklist:

- [x] random seed is set if using random data
- [x] file has been moved from `unconverted/x/y.md` to `x/y.md`
- [x] old boilerplate at top and bottom of file has been removed
- [x] Every example is independently runnable and is optimized for short line count
- [x] no more `plot()` or `iplot()`
- [x] `graph_objs` has been renamed to `graph_objects`
- [x] `fig = <something>` call is high up in each example
- [x] minimal creation of intermediate `trace` objects
- [x] liberal use of `add_trace` and `update_layout`
- [x] `fig.show()` at the end of each example
- [x] `px` example at the top if appropriate
- [x] minimize usage of hex codes for colors in favour of those in https://github.com/plotly/plotly.py-docs/issues/14

### CODE REVIEW

**Comment by emmanuelle** [comment]
The CI is not happy for the obvious reason that this tutorial will only be of use after the release, but if one of you @jonmmease or @nicolaskruchten has the time to review it then we can merge just after the release and link to this tutorial in the release communication.

**Comment by emmanuelle** [comment]
It's green now with the release, so ready for a review I think @nicolaskruchten

File: python/line-and-scatter.md [hunk 1]
```diff
@@ -40,11 +40,19 @@ jupyter:
 
 ## Scatter plot with plotly express
 
-[Plotly Express](../plotly-express/) functions take as a first argument a [tidy `pandas.DataFrame`](https://www.jeannicholashould.com/tidy-data-in-python.html). With ``px.scatter``, each data point is represented as a marker point, which location is given by the `x` and `y` columns.
+With ``px.scatter``, each data point is represented as a marker point, which location is given by the `x` and `y` columns.
```
Comment (nicolaskruchten) [inline python/line-and-scatter.md]: does the double-backtick work?

File: python/px-arguments.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+import plotly.express as px
+iris = px.data.iris()
+fig = px.scatter(iris, x=iris.sepal_length, y=iris.sepal_width, size=iris.petal_length,
+                 hover_data=[iris.index])
+fig.show()
+```
+
+### Columns not in the data_frame argument
+
+In the addition to columns from the `data_frame` argument, one may also pass columns from a different DataFrame, *as long as all columns have the same width*. It is also possible to pass columns without passing the `data_frame` argument.
```
Comment (nicolaskruchten) [inline python/px-arguments.md]: i would say "same length" rather than "same width"?

**Comment by nicolaskruchten** [comment]
:dancer: once minor comments are addressed!

File: python/line-and-scatter.md (hunk 1 above)
Comment (nicolaskruchten) [inline python/line-and-scatter.md]: hmm we probably need to change a bunch of different doc files where we said "PX accepts a tidy dataframe"

File: python/px-arguments.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+  plotly:
+    description: Arguments accepted by plotly express functions
+    display_as: file_settings
+    has_thumbnail: true
+    ipynb: ~notebook_demo/253
+    language: python
+    layout: user-guide
+    name: Plotly Express Arguments
+    order: 1
+    page_type: example_index
```
Comment (nicolaskruchten) [inline python/px-arguments.md]: this should actually be `page_type: u-guide` I think

File: python/px-arguments.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+    version: 3.7.3
+  plotly:
+    description: Arguments accepted by plotly express functions
+    display_as: file_settings
+    has_thumbnail: true
+    ipynb: ~notebook_demo/253
+    language: python
+    layout: user-guide
+    name: Plotly Express Arguments
+    order: 1
```
Comment (nicolaskruchten) [inline python/px-arguments.md]: this should be a bit further down than 1 :)

**Comment by nicolaskruchten** [comment]
P.S. this is great documentation, thank you!

File: python/px-arguments.md (hunk 2 above)
Comment (emmanuelle) [reply python/px-arguments.md]: oh sure it's a typo

File: python/line-and-scatter.md (hunk 1 above)
Comment (emmanuelle) [reply python/line-and-scatter.md]: Looks like it works... but it's from my rest habits, single ones are better

**Comment by emmanuelle** [comment]
I removed a couple of "tidy dataframes" but then I got tired and I don't want to hold this one back, so I opened #147

**Pull request merged**